timeboost-sequencer/src/lib.rs

```diff
@@ -218,27 +218,44 @@ impl Task {
             select! {
                 result = self.sailfish.next() => match result {
                     Ok(actions) => {
-                        let mut payloads: BTreeMap<RoundNumber, Vec<CandidateList>> = BTreeMap::new();
```

Comment:
As soon as a quorum of parties has reached consensus, they will produce deliver actions. However the candidate lists of late parties may appear in later iterations. In order to guarantee strictly monotonic round numbers all consecutive deliver actions need to be passed to the inclusion phase, together with their highest round number. This will put the late arrivals together with the next round quorum candidates. The previous implementation would have passed the late candidate list(s) alone to the includer, potentially producing an inclusion list with the same round number.